### ISSUE TITLE
FEAT: Make it possible to reorder and group Saved Queries

### ISSUE THREAD

**Issue by audiomuze** [beekeeper-studio/beekeeper-studio#475]
**Is your feature request related to a problem? Please describe.**
One thing I've always battled with when using SQLite is the ability to store and access queries in a user friendly, easily accessible way.  beekeeper-studio provides that, however, queries are all saved in the order created so it makes ordering discrete queries that may be used standalone or in sequence difficult.  It's also not possible to group saved queries by topic, database etc.

**Describe the solution you'd like**
It'd be really useful if Saved Queries could 1) be ordered by dragging and dropping; and 2) grouped by database or a user assigned name/project

**Comment by rathboma** [comment]
Yes! We have plans to do this for sure. With folders too. I'll mark this as accepted, but it's probably not going to get done until some time in 2021
